packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -597,9 +546,15 @@ export class MergeTree {
 		}
 	}
 
-	// TODO: remove id when segment removed
-	public mapIdToSegment(id: string, segment: ISegment) {
-		this.idToSegment.set(id, segment);
+	public unlinkMarker(marker: Marker) {
+		const id = marker.getId();
+		if (id) {
+			this.idToMarker.delete(id);
+		}
+	}
+
+	private mapIdToMarker(id: string, marker: Marker) {
```

Comment:
this should just be inlined now that it's private, the implementation is IMO more clear than the member name.